docker/1.0-1/final/Dockerfile.cpu

```diff
@@ -54,4 +54,4 @@ ENV TEMP=/home/model-server/tmp
 
 # Required label for multi-model loading
 LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
-
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
```

Comment:
can you document why this is needed now?

Reply:
Reverted the change. Upon additional testing I found out that this was not needed.